setup.py

```diff
@@ -392,6 +396,15 @@ def run(self):
 with open(requirements_path) as f:
     install_requires = f.read().splitlines()
 
+
+if is_manylinux:
+    AUDITWHEEL_PLAT = environ.get('AUDITWHEEL_PLAT', None)
+    if AUDITWHEEL_PLAT == 'manylinux2014_aarch64':
+        for i in range(len(install_requires)):
+            req = install_requires[i]
+            if req.startswith("numpy"):
+                install_requires[i] = "numpy >= 1.19.5"
```

Comment:
Don't fully follow this. We've requirements.txt that has numpy >= 1.16.6. And then we've install_requires that has this. Which one takes precedence? This has direct user impact as our pkg will fail to install with this new change.

Reply:
requirements.txt  is only a normal text file that consumed by this script only.  This script:
1. Read the text file
2. Replace numpy requirements if this build is for Linux ARM, otherwise keep it unchanged. 

The second step is needed because numpy doesn't provide ARM packages in version 1.16.x. If we really want the version then we need to build numpy from source. And this needs be done in an emulator and takes hours. Previously I manually did it and upload the result binary to azure blob storage. But it is hard to maintain. If I'm not here, nobody else knows how to do it. So if we increase the version, it's all simpler. And it only impact Linux ARM packages. And normally an end user won't build numpy from source on a small device too.